controller/transfer/knowledge_base_transfer_manager.py

```diff
@@ -11,6 +11,8 @@ def import_knowledge_base_file(project_id: str, task: UploadTask) -> None:
     task_manager.update_task(
         project_id, task.id, state=enums.UploadStates.PENDING.value
     )
+    general.commit()
```

Comment:
why was this added?

Reply:
Because of a rollback in case of an exception this would be rollbacked too if not commited

Reply:
but inside the function there it's added with the with_commit flag so it's committed anyway, right?

Reply:
yes, if we trust that this will not be changed sometimes than it can be removed here